src/test/resources/alma-fix/990016782920206441.json

```diff
@@ -158,6 +178,10 @@
       "id" : "http://lobid.org/organisations/DE-62#!",
       "label" : "Stadtbibliothek Wuppertal"
     },
+    "inCollection" : [ {
+      "id" : "http://lobid.org/organisations/7816#!",
```

Comment:
This is no valid organisation.

Reply:
I talked to @acka47 this is `Bibliotheken ohne zentrale Katalogisierung	49HBZ_BRIDGE_FIZ	7816` it has no ISIL an therefore no entry in lobid-organisations. I have introduced a dummy uri now: http://lobid.org/organisations/BRIDGE_FIZ-7816#! so we can identify this bridge.